src/Sulu/Bundle/AdminBundle/Tests/Unit/Metadata/SchemaMetadata/SingleSelectionPropertyMetadataMapperTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Tests\Unit\Metadata\SchemaMetadata;
+
+use PHPUnit\Framework\TestCase;
+use Sulu\Bundle\AdminBundle\Metadata\SchemaMetadata\SingleSelectionPropertyMetadataMapper;
+use Sulu\Component\Content\Metadata\PropertyMetadata;
+
+class SingleSelectionPropertyMetadataMapperTest extends TestCase
+{
+    /**
+     * @var SingleSelectionPropertyMetadataMapper
+     */
+    private $singleSelectionPropertyMetadataMapper;
+
+    protected function setUp(): void
+    {
+        $this->singleSelectionPropertyMetadataMapper = new SingleSelectionPropertyMetadataMapper();
+    }
+
+    private function getNullSchema(): array
+    {
+        return [
+            'type' => 'null',
+        ];
+    }
+
+    public function testMapPropertyMetadata(): void
+    {
+        $propertyMetadata = new PropertyMetadata();
+        $propertyMetadata->setName('property-name');
+
+        $jsonSchema = $this->singleSelectionPropertyMetadataMapper->mapPropertyMetadata($propertyMetadata)->toJsonSchema();
+
+        $this->assertEquals([
+            'name' => 'property-name',
+            'anyOf' => [
+                $this->getNullSchema(),
```

Comment:
Any specific reason to extract that? It's only used once, the next two lines would have to be extracted then as well, and writing this line is longer than the content the function returns... I think I would inline it.

Reply:
In here it's used just once, right. But in some other classes (like `SelectionPropertyMetadataMapperTest`) it's used more often and I think it makes the code a lot more readable. And for consistency I thought I'm gonna add this utility function to all tests needing a `null`-schema

Reply:
Wouldn't have done it in the other places either, but I don't care about it too much, so it is ok for me.